rfcs/packaging/package-structure.md

```diff
@@ -0,0 +1,145 @@
+# RFC: Package structure during convergence
+
+**Note that this RFC does not cover folder structure within the repo, most aspects of allowed import paths, build output structure, or anything about non-converged packages besides `@fluentui/react`.**
+
+**[Related RFC around near-term steps for communicating package convergence status](https://github.com/microsoft/fluentui/pull/16772)**
+
+## Summary
+
+<!-- TODO once more open issues are settled -->
+
+## Problem statement
+
+We need initial plans for these closely-related issues:
+
+- How people should consume converged components once they're ready
+- How converged components should be promoted to `@fluentui/react` (if at all) and impact on `@fluentui/react` versioning
+- How to retire legacy components
+
+### Background: current packages
+
+See the [communicating convergence status](https://github.com/microsoft/fluentui/pull/16772) RFC for details of the current package structure.
+
+## Detailed Design or Proposal
+
+**The things proposed here will not be implemented before version 8 release.**
+
+### Suite package(s) for only converged components
+
+While consumers are welcome to use individual converged component packages directly, we'll also add one or more suite package(s) containing only converged components.
+
+See open issues section for details.
+
+### Requirements for promotion to `@fluentui/react`
+
+_(The following assumes `@fluentui/react` continues to export legacy components, as well as eventually converged components--this is likely, but see open issues around suite packages.)_
+
+At some point, the goal is (probably) to export converged components from `@fluentui/react` in a way that facilitates a smooth transition for existing partners. Requirements:
+
+- APIs are stable
+- We have tooling to help with the API migration (if any)
+- All references within the suite are updated
+  - This gets harder when props of the component are exposed within other components' props (like `buttonProps: IButtonProps`)
+  - Primary motivation for this requirement: bundle size
+
```

Comment:
Quick question here, are we developing the new converged components based on the design specs' new styling? If so, what's going to happen when we mix and match with the legacy components?

Reply:
Good question, though I'm probably not the right person to answer it. @xugao do you have any thoughts here?

Reply:
> are we developing the new converged components based on the design specs' new styling

Yes, we are.

> what's going to happen when we mix and match with the legacy components?

Great question. I don't know :( @JustSlone any thought?

Reply:
@xugao, @khmakoto, @ecraig12345 answered on chat